captum/attr/_core/integrated_gradients.py

```diff
@@ -37,142 +53,143 @@ def attribute(
         return_convergence_delta=False,
     ):
         r"""
```

Comment:
Indented the documentation under `r"""`

Reply:
Yeah, I noticed that also in other algorithms. So we can adjust the indention in other methods as well. Docstring will be consistently aligned with r""". What do you think  ?